### PR TITLE
16.0 add sale channel

### PR DESCRIPTION
**Pull request by sebastienbeau**
Extract existing module from akretion repo

sale-channel

### CODE REVIEW

**Review by sbejaoui — APPROVED** [review]

**Review by sbejaoui — APPROVED** [review]
code review + functional test

File: sale_channel/models/sale_channel.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+from odoo import fields, models
+
+
+class SaleChannel(models.Model):
+    _name = "sale.channel"
+    _description = "Sale Channel"
+
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
```
Comment (bealdav) [inline sale_channel/models/sale_channel.py:12]: Seems boolean fields are True, by default !?

File: sale_channel/models/sale_channel.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    partner_id = fields.Many2one("res.partner")
```
Comment (hparfr) [inline sale_channel/models/sale_channel.py]: who is this partner ?

File: sale_channel/models/sale_channel.py (hunk 1 above)
Comment (sebastienbeau) [reply sale_channel/models/sale_channel.py:12]: no, boolean are False by default

**Comment by sebastienbeau** [comment]
let's merge it ?

**Comment by sebastienbeau** [comment]
@hparfr @lmignon

**Review by hparfr — APPROVED** [review]

**Review by bealdav — APPROVED** [review]

**Pull request merged**